### ISSUE TITLE
netty-tcnative-parent-2.0.5.Final build issue

### ISSUE THREAD

**Issue by dineshappavoo** [netty/netty-tcnative#308]
I am trying to build netty-tcnative-parent-2.0.5.Final with openssl-static. I am getting build error while fetching apr 1.5.2

```

[INFO] --- maven-antrun-plugin:1.8:run (build-apr-linux-mac) @ netty-tcnative-openssl-static ---
[INFO] Executing tasks

main:
      [get] Getting: http://www.us.apache.org/dist/apr/apr-1.5.2.tar.gz
      [get] To: /Users/ad/Documents/docs/ssl/openssl/version-2-0-5/netty-tcnative/openssl-static/target/apr-1.5.2.tar.gz
      [get] Error opening connection java.io.FileNotFoundException: http://www.us.apache.org/dist/apr/apr-1.5.2.tar.gz
      [get] Error opening connection java.io.FileNotFoundException: http://www.us.apache.org/dist/apr/apr-1.5.2.tar.gz
      [get] Error opening connection java.io.FileNotFoundException: http://www.us.apache.org/dist/apr/apr-1.5.2.tar.gz
      [get] Can't get http://www.us.apache.org/dist/apr/apr-1.5.2.tar.gz to /Users/ad/Documents/docs/version-2-0-5/netty-tcnative/openssl-static/target/apr-1.5.2.tar.gz
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
```

It looks like pom file has apr `1.5.2` version [ https://github.com/netty/netty-tcnative/blob/netty-tcnative-parent-2.0.5.Final/pom.xml#L52 ] from `http://www.us.apache.org/dist/apr/${aprArchiveFile}` URL. [ ref: https://github.com/netty/netty-tcnative/blob/7a72db574db3b4df20d48b5092baa633aa773964/pom.xml#L322 ]. 

But this version is not available in the apache distribution http://www.us.apache.org/dist/apr/ (atleast in this URL).

I am not sure there is fix for this to use different apache dist URL. But when I changed the apr version to 1.6.2 in netty-tcnative/pom.xml , it worked. I wanted to notify the team.

**Comment by floragunncom** [comment]
See https://mail-archives.apache.org/mod_mbox/apr-dev/201710.mbox/%3cCACsi2532W69+QrKoYUwB--pqOF1f_OfmzY6A8u2AybsWYVdDyg@mail.gmail.com%3e

**Comment by lagrangian-mech** [comment]
Since archive.apache.org appears to have both the current version and past versions, perhaps the pom should be changed from http://www.us.apache.org/... to http://archive.apache.org/...

Then both builds of the latest version of netty-tcnative and past versions will be able to find what they need.

**Comment by normanmaurer** [comment]
@lagrangian-mech https://github.com/netty/netty-tcnative/pull/314

**Comment by normanmaurer** [comment]
Fixed

**Comment by dineshappavoo** [comment]
@normanmaurer Thanks for taking care of this. I think the next release `2.0.7` should be fine with this change. But for the current release 2.0.6 it is failing now because apache released a new version 1.6.3 and they removed 1.6.2 from http://www.us.apache.org/dist/apr/ . 

I understand your change is in master. Is it possible to update your change at least in the current release 2.0.6.Final? It would be really helpful, because we run a CI to build openssl-static. So that we don't have to add a hack to update the apr version only for this release.

**Comment by normanmaurer** [comment]
@dineshappavoo its a bad-practise to change code in a tag that was released. We will release 2.0.7.Final this week which will have the fix.
